Mage.Sets/src/mage/cards/r/RazeToTheGround.java

```diff
@@ -0,0 +1,82 @@
+package mage.cards.r;
+
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CantBeCounteredSourceEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetArtifactPermanent;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class RazeToTheGround extends CardImpl {
+
+    public RazeToTheGround(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{2}{R}");
+
+        // This spell can't be countered.
+        Effect effect = new CantBeCounteredSourceEffect();
+        effect.setText("this spell can't be countered");
+        Ability ability = new SimpleStaticAbility(Zone.STACK, effect);
+        ability.setRuleAtTheTop(true);
+        this.addAbility(ability);
```

Comment:
This can just use `CantBeCounteredSourceAbility`.  No need to set text either.  That's its default text.

Reply:
Fixed for this card and others using the same code

Reply:
I think these still need `ability.setRuleAtTheTop(true);` unless you've tested and found otherwise (database likely needs to be cleared when you're testing text changes so it updates with new text).

Reply:
Checked in game and reapplied to cards that should have had it